frontend/src/App.js

```diff
@@ -13,11 +16,21 @@ function App() {
         },
         [])
 
+    const onDraftUpdated = (updatedDraft) => {
+        setDraft(updatedDraft);
+    }
 
     return (
         <div>
-            <RecipeList recipes={recipes} />
-            <FileUpload/>
+            <Switch>
+                <Route exact path="/">
+                    <RecipeList recipes={recipes}/>
+                    <FileUpload onDraftReceived={onDraftUpdated}/>
+                </Route>
+                <Route path="/draft">
+                    { draft !== '' && <RecipeDraft recipe={draft} onRecipeUpdated={onDraftUpdated} /> }
```

Comment:
Why dont u use `{draft && ... }` ? As an empty string evaluates to false, everything should work as expected.